File: src/main/java/org/example/coin_laundry_app_backend/notification/domain/model/value/NotificationMessage.java
package org.example.coin_laundry_app_backend.notification.domain.model.value;

import lombok.Getter;

// TODO: 전화번호 인증 구현을 위한 임시 구현
@Getter
public class NotificationMessage {

    private final String msgCd;
    private final String to;
    private final String[] params;

    public NotificationMessage(String msgCd, String to, String[] params) {
        this.msgCd = msgCd;
        this.to = to;
        this.params = params;
    }
}

File: carry-notification/src/main/kotlin/com/carry/notification/domain/vo/NotificationMessage.kt
package com.carry.notification.domain.vo

/**
 * 알림 메시지 본문(제목·내용). 함께 채워지는 응집 개념이라 하나의 VO로 묶는다.
 */
data class NotificationMessage(
    val title: String,
    val content: String,
)
